workshops/models.py

```diff
@@ -271,14 +277,14 @@ def uninvoiced_events(self):
 class Event(models.Model):
     '''Represent a single event.'''
 
-    published  = models.BooleanField(default=False)
     site       = models.ForeignKey(Site)
     tags       = models.ManyToManyField(Tag)
     organizer  = models.ForeignKey(Site, related_name='organizer', null=True, blank=True)
     start      = models.DateField(null=True, blank=True)
     end        = models.DateField(null=True, blank=True)
     slug       = models.CharField(max_length=STR_LONG, null=True, blank=True)
-    url        = models.CharField(max_length=STR_LONG, unique=True, null=True, blank=True)
+    url        = models.CharField(max_length=STR_LONG, unique=True, null=True, blank=True,
+                                  help_text='Setting this "publishes" the event.')
```

Comment:
It's [a bit more complicated][1] than just setting `Event.url`.  You should also document the effect of `Event.start` (both need to be set for publication).

[1]: https://github.com/swcarpentry/amy/pull/363/files#r32372570